### PR TITLE
Mldb 1621 support and test weights usage in classifier.test

### PR DESCRIPTION
**Pull request by simlmx**
- add a general test for classifier.test
   - right now it tests either the output dataset OR the stats returned by the query, whichever let me test that the weights work easily
- make sure it supports weights for all modes (boolean, regression, categorical)
- minor related cleaning and fixes

### CODE REVIEW

File: testing/test_classifier_test_proc.py [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+            'params': {
+                'testingData': "SELECT score, label:{}, weight:{} FROM ds".format(label, weight),
+                'outputDataset': 'out',
+                'mode': mode
+            }
+        }
+
+    def test_boolean_dataset_no_weight(self):
+        res = mldb.post('/v1/procedures', self._get_params('boolean', 'bool_label', '1')).json()
+        # TODO test the output of res
```
Comment (guyd) [inline testing/test_classifier_test_proc.py]: I believe this is already done.

**Comment by guyd** [comment]
Will you update the document accordingly?

File: plugins/accuracy.cc [hunk 2]
```diff
@@ -225,15 +225,15 @@ runBoolean(AccuracyConfig & runAccuracyConf,
         output->commit();
     }
 
-    cerr << "stats are " << endl;
+    // cerr << "stats are " << endl;
 
-    cerr << stats.toJson();
+    // cerr << stats.toJson();
```
Comment (mailletf) [inline plugins/accuracy.cc]: je laisserais celui la. je m'en sers souvent :P

Comment (simlmx) [inline plugins/accuracy.cc]: Bof c'est ça qui est retourné ensuite. Utilise ce qui est retourné non? Moi je trouve que ça me pollue mon output!

**Pull request merged**